Sources/Administrator/InputHandler.cpp

```diff
@@ -0,0 +1,121 @@
+//
+// Created by daria on 21.05.16.
+//
+
+#include "InputHandler.h"
+#include <iostream>
+#include <cstring>
+
+void InputHandler::run()
+{
+    std::cout << "Waiting for a command:\n";
+    std::string command;
+    std::string name;
+
+    while ( name != "exit")
```

Comment:
Maybe do while loop would be better? Just suggestion, but i think in this case code is better structured in `do {} while`